### DISCUSSION TITLE
建议将主分支切换到PySide6

### DISCUSSION THREAD

**Discussion by H1DDENADM1N** [ChinaIceF/PyQt-SiliconUI#25] [Q&A] [answered]
PySide6作为Qt6的官方Python绑定，具有以下优势：

1. 官方维护和支持：由Qt公司直接维护，确保了PySide6的稳定性和长期支持，用户可以期待定期的更新和补丁。

2. Qt6的最新功能：PySide6提供了对Qt6最新功能和模块的访问，包括性能改进、新的API和增强的跨平台支持。

3. API一致性：PySide6的API与Qt6的C++ API非常相似，这使得C++开发者可以更容易地转移到Python，并且便于在两种语言之间进行代码迁移。

4. 社区和资源：随着Qt6的推出和PySide6的官方支持，Python开发者社区对PySide6的兴趣和参与度正在增长，这意味着更多的资源、教程和第三方库可能会被开发出来。

5. Shiboken6：PySide6使用Shiboken6作为其生成Python绑定的工具，这是Shiboken的更新版本，提供了改进的性能和更好的集成。

6. 未来发展：随着Qt的不断更新和发展，PySide6也将继续更新以保持与Qt的同步，为Python开发者提供最新的Qt功能。

综上所述，PySide6为Python开发者提供了一个现代化、高性能且易于使用的GUI开发工具，它是比PyQt5更有吸引力的选择。

因此，建议将主分支切换到PySide6。

另，PySide6 6.7.* 新版本bug较多，建议保持 PySide6<=6.6.3.1

**Comment by H1DDENADM1N** [comment]
### 我将 [PySide6-SiliconUI](https://github.com/H1DDENADM1N/PySide6-SiliconUI) 分支同步更新到了 `v1.0.1`

**Comment by rainzee** [comment, marked as answer]
做为一个维护过多个 Qt 组件库，给上游 Qt 提交过 bug 写过代码的，其实 Widgets 平台存在的意义大部分情况下已经从传统 Desktop GUI 领域到了，工控机，医疗，航天等领域，该领域还存在大量的旧设备，需要 Qt5 的支持，目前最好的 Qt5 绑定就是 PyQt5，且由于你可能没有实际的工程经验，其实 PySide6 存在诸多问题，比较严重的是多媒体模块的内存泄漏和释放问题。组件库目前处在重构和设计阶段，将在很长一段时间内以 Qt5 为主导，PySide6 的绑定支持非常简单，只是我们要分清楚优先级，以上。

**Reply by H1DDENADM1N** [reply]
非常漂亮的界面库，希望“很长一段时间”不会太长。🥰

另，望您也解答下 讨论#26 [使用 QtPy 统一 PyQt5 和 PySide6 分支](https://github.com/ChinaIceF/PyQt-SiliconUI/discussions/26)
